cranelift/codegen/src/loop_analysis.rs

```diff
@@ -239,7 +241,8 @@ impl LoopAnalysis {
                 cfg.pred_iter(self.loops[lp].header)
                     .filter(|pred| {
                         // We follow the back edges
-                        domtree.dominates(self.loops[lp].header, pred.inst, layout)
+                        let pred_block = layout.inst_block(pred.inst).unwrap();
```

Comment:
No need to use the `inst_block` method here -- `pred.block` is available (as shown a few lines below).

Likewise above around line 205.